### ISSUE TITLE
Cover broken

### ISSUE THREAD

**Issue by jomigo96** [ekspek/ist-relatorio#10]
Cover commands 
```tex
\subtitle{}
\subject{}
\course{}
\groupno{}
```
not working in the current version: b6ea808a387e7aa4a608f42aeff7e0aeaa69e69f

**Comment by ekspek** [comment]
I updated most commands to avoid conflicts with basic `article` class commands, which consisted of changing most commands of this class (except `author` and `date`) to similar variants starting with `set`. So the commands you listed above are now written like
```tex
\setsubtitle{}
\setsubject{}
\setcourse{}
\setgroupno{}
```
The same applies to the commands `title` and `institution`, which are now also written like
```tex
\settitle{}
\setinstitution{}
```
`title`, `author` and `date` still work because they are basic `article` class macros. `title` already has an `ist-report` equivalent, while the other two do not for the time being.

I already updated the documentation for these matters but forgot to compile and include the PDF itself (my bad).
